spec/features/remember_device/totp_spec.rb

```diff
@@ -23,8 +23,20 @@ def remember_device_and_sign_out_user
   end
 
   context 'sign up' do
+    it 'does not allow choosing totp as backup auth method after it is used as primary' do
```

Comment:
Why is this in the remember device specs? It does not look related to remember device? I think you may want to create a separate spec file for this? Maybe something like `spec/features/mfa_enrollability_spec.rb` or something.

Reply:
Weird.  That was the only feature spec for totp.  Yes I can move it.